util/db/helmrepository.go

```diff
@@ -15,7 +15,9 @@ import (
 
 func getHelmRepoCredIndex(s *settings.ArgoCDSettings, repoURL string) int {
 	for i, cred := range s.HelmRepositories {
-		if strings.ToLower(cred.URL) == strings.ToLower(repoURL) {
+		// work-around the SA6005 check as I'm not sure that the suggested changes is correct
```

Comment:
Found description of SA6005 https://staticcheck.io/docs/checks#SA6005. It suggests using `strings.EqualFold(s1, s2).`